scraper/debian.py

```diff
@@ -22,59 +22,26 @@
 #  Visit https://github.com/nexB/vulnerablecode/ for support and download.
 
 import logging
-import re
 from urllib.request import urlopen
+import json
 
 import bs4
 
+DEBIAN_ROOT_URL = "https://security-tracker.debian.org/tracker/data/json"
 
-DEBIAN_ROOT_URL = 'https://security-tracker.debian.org'
 
-
-def extract_tracker_paths(html):
-    """
-    Return a list of tracker URL paths extracted from the given `html` input.
-    """
-    soup = bs4.BeautifulSoup(html, 'lxml')
-    tracker_links = soup.findAll('a', href=re.compile('^/track+.*'))
-    return [link.get('href') for link in tracker_links]
-
-
-def extract_cves_from_tracker(html):
+def debian_extract_data():
     """
     Return all CVEs extracted from the given `html` input.
     """
-    cve_id = []
-    package_name = []
-    vulnerability_status = []
-    soup = bs4.BeautifulSoup(html, 'lxml')
-
-    for tag in soup.find_all('a'):
-        href = tag.get('href')
-
-        if re.search('/tracker/CVE-(.+)', href):
-            id = re.findall('(?<=/tracker/).*', href)
-            cve_id.append(id[0])
-
-        if re.search('^/tracker/TEMP-+.*', href):
-            id = re.findall('(?<=/tracker/).*', href)
-            cve_id.append(id[0])
-
-        if re.search('/tracker/source-package/(.+)', href):
-            pkg = re.findall('(?<=/tracker/source-package/).*', href)
-            package_name.append(pkg[0])
-
-        # if package name is empty, use the previous package name
-        if href == '/tracker/source-package/':
-            package_name.append(pkg)
+    test_input = urlopen("https://security-tracker.debian.org/tracker/data/json").read()
```

Comment:
you need to load this with the `json` module...
e.g. once you have read the content, call `debian_vulnerabilities = json.loads(test_input)`